Android--EN/domain/RectangleConfig.java

```diff
@@ -40,8 +40,8 @@ public RectangleConfig(Double width, Double height, AnimationMode mode) {
 
     /**
      * The `RectangleConfig` constructor.
-     * <p>
-     * In this method, pass in the `width`, `height`, and `mode` parameters. Based on the `width` and `height` you pass in, the SDK calculates `originX` and `originY`, the X and Y coordinates of the top left corner of the view rectangle in the world coordinate system, in the following ways:
+     * <p> // TODO William Why do you use paragraph tags sometimes but not others? Does it matter? Is there some variation in output?
```

Comment:
This is also a result of automatic formatting. The tags have the same effect as a blank line.  The output will look the same.